doc/en/source/k8sIntegrationYAML.rst

```diff
@@ -28,259 +28,147 @@ Inorder to setup the components, please refer the steps below:
-    roleRef:
-    apiGroup: rbac.authorization.k8s.io
-    kind: ClusterRole
-    name: fogflow-dns-role
-    subjects:
-    - kind: ServiceAccount
-    namespace: fogflow  //Edit this as per previous step , for example namespace : fogflow
-    name: fogflow-dns-role
-
-**Step 5** : Configure config.json as per user's environnment like shown below:
-
-- **my_hostip**: this is the IP of your host machine, which should be accessible for both the web browser on your host machine and docker containers. Please DO NOT use "127.0.0.1" for this.
-- **site_id**: each FogFlow node (either cloud node or edge node) requires to have a unique string-based ID to identify itself in the system;
-- **physical_location**: the geo-location of the FogFlow node;
-- **worker.capacity**: it means the maximal number of docker containers that the FogFlow node can invoke;  
-
-.. code-block:: console
-
-    "my_hostip": "172.30.48.24", //User should update the IP as per his/her environment
-    "physical_location":{
-        "longitude": 139.709059,
-        "latitude": 35.692221
-    },
-    "site_id": "001",
-    "worker": {
-    "container_autoremove": false,
-    "start_actual_task": true,
-    "capacity": 8
-    }
-
-**Step 6** : Edit the namespace, serviceaccount value and configjson in dgraph-deployment.yaml as per user's environment and use below command to launch the deployments.
-
-.. code-block:: console
-
-    $kubectl create -f dgraph-deployment.yaml 
-
-**Step 7** : Edit the namespace, serviceaccount value and configjson path in discovery.yaml as per user's environment and use below command to launch the deployments.
-
-.. code-block:: console
-
-    $kubectl create -f discovery.yaml 
-
-**Step 8** : Edit the namespace, serviceaccount value and configjson path in cloud-broker.yaml as per user's environment and use below command to launch the deployments.
-
-.. code-block:: console
-
-    $kubectl create -f cloud-broker.yaml 
-
-**Step 9** : Edit the namespace, serviceaccount value and configjson path in designer.yaml as per user's environment and use below command to launch the deployments.
-
-.. code-block:: console
-
-    $kubectl create -f designer.yaml 
-    
-**Step 10** : Edit the namespace, serviceaccount value and nginxConf path in nginx.yaml as per user's environment and use below command to launch the deployments.
-
-.. code-block:: console
-
-    $kubectl create -f nginx.yaml 
-
-**Step 11** : Edit the namespace, serviceaccount value and configjson path in rabbitmq.yaml as per user's environment and use below command to launch the deployments.
+    namespace: fogflow                      
+    name: nginx
+    labels:
+        run: nginx
+    spec:
+    type: LoadBalancer
+    ports:
+        - port: 80
+        targetPort: 80
+    selector:
+        run: nginx
+    externalIPs: [172.30.48.24]  //edit this
+   
+**Step 4** : Now, edit **path** variable in dragph.yaml as per user's environment. The complete path is to be mentioned as shown below:
```

Comment:
remove now, better to write second part as : please mention the complete path as shown below:

Reply:
Comment incorporated